src/index.js

```diff
@@ -195,28 +195,16 @@ function MarkdownEditor(props) {
    * - Set a lockText annotation on the editor equal to props.lockText
    */
   useEffect(() => {
-    if (!markdownMode) {
-      if (editorRef && editorRef.current) {
-        const editor = editorRef.current;
-        const { annotations, selection } = editor.value;
-
-        editor.withoutSaving(() => {
-          annotations.forEach((ann) => {
-            if (ann.type === 'lockText') {
-              editor.removeAnnotation(ann);
-            }
-          });
-
-          if (lockText) {
-            // it doesn't matter where we put the annotation
-            // so we use the current selection
-            const annotation = {
-              key: `lockText-${uuidv4()}`,
-              type: 'lockText',
-              anchor: selection.anchor,
-              focus: selection.focus,
-            };
-            editor.addAnnotation(annotation);
+    if (props.lockText && editorRef && editorRef.current) {
```

Comment:
This seems to be picking the wrong version? Shouldn't the version from master be kept?